jest.config.js

```diff
@@ -16,4 +16,9 @@ module.exports = {
     'jest-watch-typeahead/filename',
     'jest-watch-typeahead/testname',
   ],
+  globals: {
+    'ts-jest': {
+      packageJson: 'package.json',
```

Comment:
[`ts-jest` assumes it can walk up from its location on disk to find `package.json`](https://github.com/kulshekhar/ts-jest/issues/852), but this is not true when operating in the yarn PnP virtual file system. So we provide the path here.